### PR TITLE
feat: implement about us modal

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes https://github.com/cheminfo/nmrium/issues/1920

### CODE REVIEW

**Comment by lpatiny** [comment]
@wadjih-bencheikh18 Can you check why there is this double line ?

image

**Comment by wadjih-bencheikh18** [comment]
> Can you check why there is this double line ?

styling fixed, thank you

File: src/component/header/Header.tsx [hunk 1]
```diff
@@ -107,12 +107,7 @@ function HeaderInner(props: HeaderInnerProps) {
...
 
   const openAboutUs = useCallback(() => {
-    modal.show(<AboutUsModal />, {
-      isBackgroundBlur: false,
-      position: positions.MIDDLE,
-      width: 500,
-      height: 480,
-    });
+    modal.show(<AboutUsModal />);
   }, [modal]);
```
Comment (targos) [inline src/component/header/Header.tsx:111]: You shouldn't use the `modal.show` API anymore. Have a look at the stories and the code in `react-science`. Basically you need a state with `useOnOff` and render the modal component in the children.

**Comment by targos** [comment]
Please also revert the changes to the package-lock file. I suggest you rebase your branch on `main` to be up-to-date.

File: src/component/header/Header.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/component/header/Header.tsx:111]: fixed thank you

**Comment by wadjih-bencheikh18** [comment]
@targos rabased

**Comment by targos** [comment]
There are test errors.

**Comment by wadjih-bencheikh18** [comment]
> There are test errors.

Sorry, I forgot about stylelint, Fixed now

**Pull request merged**